src/application_config/app_config.h

```diff
@@ -18,29 +47,67 @@
  * @addtogroup SDK15
  */
 /** @{ */
-/**
- * @file app_config.h
- * @author Otso Jousimaa <otso@ojousima.net>
- * @date 2020-04-16
- * @copyright Ruuvi Innovations Ltd, license BSD-3-Clause.
- *
- */
+
+#ifndef APP_FW_NAME
+#   define APP_FW_NAME "Ruuvi FW"
+#endif
+
+#ifndef APP_FW_VERSION
+#define APP_FW_VERSION  "v3.0.0"  // Overridden by 'make' provided define: -DAPP_FW_VERSION ; from git describe --tags
+#endif
+
+/* @brief Configure Logging  i.e. history * * * * * * * * * * * * * * * * * * * * */
+//   The values sent from history log are:
+#ifndef APP_LOG_TEMPERATURE_ENABLED
+#define APP_LOG_TEMPERATURE_ENABLED true
+#endif
+#ifndef APP_LOG_HUMIDITY_ENABLED
+#define APP_LOG_HUMIDITY_ENABLED    true
+#endif
+#ifndef APP_LOG_PRESSURE_ENABLED
+#define APP_LOG_PRESSURE_ENABLED    true
+#endif
+#ifndef APP_LOG_X_ENABLED
+#define APP_LOG_X_ENABLED           false
+#endif
+#ifndef APP_LOG_Y_ENABLED
+#define APP_LOG_Y_ENABLED           false
+#endif
+#ifndef APP_LOG_Z_ENABLED
+#define APP_LOG_Z_ENABLED           false
+#endif
+#ifndef APP_LOG_MOVEMENT_ENABLED
+#define APP_LOG_MOVEMENT_ENABLED    false
+#endif
+#ifndef APP_LOG_BATTVOLT_ENABLED
+#define APP_LOG_BATTVOLT_ENABLED    false
+#endif
+
+#ifndef APP_LOG_INTERVAL_S
+#define APP_LOG_INTERVAL_S                  (5u * 60u)                           // 5 MINUTES!
+#endif
+
+// Not yet implemented 3.31.1 2021-12-14
+#ifndef APP_LOG_OVERFLOW          //<! When history log fills flash: OVERFLOW true wrap around i.e. save most recent readings
+#define APP_LOG_OVERFLOW true     //                                          false  save oldest reading and loose recent ones
+#endif
+
 
 /** @brief enable nRF15 SDK implementation of drivers */
-#define RUUVI_NRF5_SDK15_ENABLED (1U)
+#define RUUVI_NRF5_SDK15_ENABLED true
 
 #ifndef APP_HEARTBEAT_OVERDUE_INTERVAL_MS
-#   define APP_HEARTBEAT_OVERDUE_INTERVAL_MS (5U * 60U * 1000U)
+#   define APP_HEARTBEAT_OVERDUE_INTERVAL_MS (5u * 60u * 1000u)
```

Comment:
Numeric suffixes are capitalized, let's not change the style for one file.

Reply:
I have tried to reduce the noise just a little by using a lowercase u since sometimes a constant like 500U can, at a glance look 10 times bigger (If the U appears to be a zero.)

Reply:
See above comment about suffix capitalization